index.js

```diff
@@ -31,35 +31,48 @@ const factoryAbi = [
   "function create() public returns (address)"
 ];
 
-const SHORT_NAME_MAINNET = "w3q";
 const SHORT_NAME_GALILEO = "w3q-g";
 const SHORT_NAME_ETHEREUM = "eth";
-const SHORT_NAME_RINKEBY = "rin";
+const SHORT_NAME_GOERLI = "gor";
+const SHORT_NAME_SEPOLIA = "sep";
+const SHORT_NAME_OPTIMISTIC = "oeth";
+const SHORT_NAME_ARBITRUM = "arb1";
+const SHORT_NAME_OPTIMISTIC_GOERLI = "ogor";
+const SHORT_NAME_ARBITRUM_GOERLI = "arb-goerli";
+const SHORT_NAME_EVMOS = "evmos";
+const SHORT_NAME_ARBITRUM_NOVE = "arb-nova";
 
-const MAINNET_NETWORK = "mainnet";
-const GALILEO_NETWORK = "galileo";
-const DEVNET_NETWORK = "devnet";
-
-const MAINNET_CHAIN_ID = 333;
 const GALILEO_CHAIN_ID = 3334;
 const ETHEREUM_CHAIN_ID = 1;
-const RINKEBY_CHAIN_ID = 4;
+const GOERLI_CHAIN_ID = 5;
+const SEPOLIA_CHAIN_ID = 11155111;
+const OPTIMISTIC_CHAIN_ID = 10;
+const ARBITRUM_CHAIN_ID = 42161;
+const OPTIMISTIC_GOERLI_CHAIN_ID = 420;
+const ARBITRUM_GOERLI_CHAIN_ID = 421613;
+const EVMOS_CHAIN_ID = 9001;
+const ARBITRUM_NOVE_CHAIN_ID = 42170;
```

Comment:
You may want checkout this newer list [here](https://github.com/QuarkChain/web3q-server/issues/26#issuecomment-1327027561)